eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -164,46 +164,62 @@ function DeployStressPackage(
         if ($LASTEXITCODE) { return }
     }
 
-    $imageTag = "${registryName}.azurecr.io"
+    $imageTagBase = "${registryName}.azurecr.io"
     if ($repositoryBase) {
-        $imageTag += "/$repositoryBase"
+        $imageTagBase += "/$repositoryBase"
     }
-    $imageTag += "/$($pkg.Namespace)/$($pkg.ReleaseName):${deployId}"
-
-    $dockerFilePath = if ($pkg.Dockerfile) {
-        Join-Path $pkg.Directory $pkg.Dockerfile
+    $imageTagBase += "/$($pkg.Namespace)/$($pkg.ReleaseName):${deployId}"
+
+    $dockerFilePaths =@()
+    
+    $genValFile = Join-Path $pkg.Dockerfile "generatedValues.yaml"
+    if (Test-Path $genValFile) {
+        $scenarios = (Get-Content $genValFile -Raw | ConvertFrom-Yaml).Scenarios
+        foreach ($scenario in $scenarios) {
+            $dockerfilePath = $scenario.image
+            $dockerFilePath = [System.IO.Path]::GetFullPath($dockerFilePath)
+            $dockerFilePaths += $dockerFilePath
+        }
```

Comment:
I think you also need to change the scenario image value to be the image tag (or ideally use different keys for each).